app/services/open_library/book_search.rb

```diff
@@ -4,16 +4,21 @@ def self.execute(isbn_number)
       isbn = "ISBN:#{isbn_number}"
       response = get('/api/books', query: { bibkeys: isbn }).parsed_response[isbn]
       if response.nil?
-        "El libro no existe"
-      else
-        {
-          isbn: isbn_number,
-          title: response['title'],
-          subtitle: response['subtitle'],
-          number_of_pages: response['number_of_pages'],
-          authors: response['authors']
-        }
+        raise ::Errors::OpenLibrary::BookNotFound.new,
+              "The book with id #{isbn_number} not found"
```

Comment:
We could use I18n for this.
But since you already saw how the translation works in #13 I don't think is necesary to implement it again. Keep in mind that many projects don't have explicit strings and you may have to use I18n for this with something like:
```suggestion
I18n.t('services.open_library.errors.book_not_found', isbn_number: isbn_number)
```
And you add the locale in the corresponding translation, for example for spanish and english you may have 2 files:
```
en:
  services:
    open_library:
      errors:
        book_not_found: 'The book with ISBN: %{isbn_number} could not be found'
```
```
es:
  services:
    open_library:
      errors:
        book_not_found: 'El libro con ISBN: %{isbn_number} no existe
```